### PR TITLE
Task/APPS-2410 — add crc dashboard and downloads

### PR DESCRIPTION
**Pull request by giuliacampana**
adds separate Dashboard and Downloads pages if CRC module is selected.

### CODE REVIEW

File: src/containers/dashboard/CrisisReportDashboardContainer.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  align-items: center;
+`;
+
+const Flex = styled.div`
+  display: flex;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+`;
```
Comment (UnsungHero97) [inline src/containers/dashboard/CrisisReportDashboardContainer.js:38]: Doesn't there exist a common component like this already?

File: src/containers/dashboard/CrisisReportDashboardContainer.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    loadDashboardData :RequestSequence;
+  }
+};
+
+type State = {
+  layout :string;
+  timeRange :any;
+};
+
+class DashboardContainer extends React.Component<Props, State> {
```
Comment (UnsungHero97) [inline src/containers/dashboard/CrisisReportDashboardContainer.js:92]: Please switch to a function components + hooks.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/containers/dashboard/CrisisReportDashboardContainer.js (hunk 1 above)
Comment (giuliacampana) [reply src/containers/dashboard/CrisisReportDashboardContainer.js:38]: There might be. I just copied all of this into a new file. I'll check to see if there's an existing component to replace this with.

File: src/containers/dashboard/CrisisReportDashboardContainer.js (hunk 2 above)
Comment (giuliacampana) [reply src/containers/dashboard/CrisisReportDashboardContainer.js:92]: this is actually legacy CARE code that i just copied to a new file—but i can do that to keep the project updated

**Review by UnsungHero97 — DISMISSED** [review]

File: src/containers/dashboard/DashboardContainer.js [hunk 3]
```diff
@@ -1,204 +1,46 @@
...
+      component = <CrisisReportDashboardContainer />;
+      break;
+    case 'crc':
+      component = <CRCDashboardContainer root={DASHBOARD_PATH} match={match} organizationId={organizationId} />;
+      break;
+    case 'helpline':
+      component = <CrisisReportDashboardContainer />;
+      break;
+    default:
+      component = <CrisisReportDashboardContainer />;
```
Comment (anzioka) [inline src/containers/dashboard/DashboardContainer.js:33]: Would have been nice to constantify these strings

Looks like this switch statement only needs two cases: `crc` and `default`

File: src/containers/downloads/DownloadsContainer.js [hunk 4]
```diff
@@ -19,11 +20,15 @@ const DownloadsContainer = () => {
...
   switch (profileModule) {
-    case 'helpline': {
+    case 'crisis':
+      component = <CrisisReportDownloadsContainer />;
+      break;
+    case 'crc':
+      component = <CRCDownloadsContainer root={DOWNLOADS_PATH} match={match} organizationId={organizationId} />;
+      break;
+    case 'helpline':
       component = <HelplineDownloadsContainer root={DOWNLOADS_PATH} match={match} organizationId={organizationId} />;
```
Comment (anzioka) [inline src/containers/downloads/DownloadsContainer.js:30]: same comment about constantifying

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**